config/default.json

```diff
@@ -1,3 +1,10 @@
 {
-    "port": 4000
+    "port": 4000,
+    "db": {
```

Comment:
This configuration is build for one DB, I'd prepare configuration for more than one DB.

Reply:
done